curricula/templates/curricula/partials/code_studio.html

```diff
@@ -16,13 +16,9 @@
                         <li class="chunk-header type-{{ level.type }} {{ unit.slug }}-stage-{{ lesson.number }}-level-{{ level.position }}-tab" data-level="{{ level.position }}" data-named="{{ level.named_level }}">
                             {{ level.display_name|default:level.name }}
                             <span class="level-icon fa"></span>
-                            <!-- TODO: Remove outer if statement here when -->
-                                    <!-- we want to roll out for all courses -->
-                                    {% if '2019' in level.path %}
```

Comment:
this is safe to remove because we've removed the assessment flag from puzzles in all CSF scripts we care about, right?

Reply:
Correct - we did the work on code studio to make sure only things we want marked as assessments are marked as assessments so when we pull the information over it should be correct.